empress/topoSort/TreeTester.py

```diff
@@ -0,0 +1,42 @@
+# TreeTester.py
```

Comment:
Move to `test/topoSort/test_Tree.py` (create a new folder called topoSort in the folder test and move this there).